modal/cls.py

```diff
@@ -879,6 +881,38 @@ def _deps():
         cls._options.merge_options(batching_options)
         return cls
 
+    def with_http_server(self: "_Cls", *, port: int, proxy_region: Literal["us-east", "us-west", "ap-south", "True"], startup_timeout: Optional[int] = None, exit_grace_period: Optional[int] = None) -> "_Cls":
```

Comment:
Is this something we need at this point? It would be exposed in the public api of Cls